blocks/api/paste/strip-attributes.js

```diff
@@ -0,0 +1,12 @@
+export default function( nodes ) {
+	// MUTATION!
+	nodes.forEach( deepAttributeStrip );
+	return nodes;
+}
+
+function deepAttributeStrip( node ) {
+	node.removeAttribute( 'style' );
+	node.removeAttribute( 'class' );
+	node.removeAttribute( 'id' );
+	Array.from( node.children ).forEach( deepAttributeStrip );
```

Comment:
Line 11 calls itself recursively so all children (and _their_ children…) lose those attributes as well